MoreLinq/RandomSubset.cs

```diff
@@ -23,6 +23,45 @@ namespace MoreLinq
 
     public static partial class MoreEnumerable
     {
+        /// <summary>
+        /// Returns a sequence of random elements from the original sequence.
+        /// </summary>
+        /// <typeparam name="T">The type of source sequence elements.</typeparam>
+        /// <param name="sequence">
+        /// The sequence from which to return random elements.</param>
+        /// <returns>
+        /// A random sequence of elements in random order from the original
```

Comment:
Random is mentioned twice which confuses the mind. Rephrase to:

> A sequence of elements in random order from the original…

Is it worth mentioning that there is no chance of duplication, unless of course, the original has duplicates in the first place?